nifi-nar-bundles/nifi-snmp-bundle/nifi-snmp-processors/src/main/java/org/apache/nifi/snmp/processors/ListenTrapSNMP.java

```diff
@@ -56,6 +61,11 @@
 @RequiresInstanceClassLoading
 public class ListenTrapSNMP extends AbstractSessionFactoryProcessor {
 
+    public static final AllowableValue USM_JSON_FILE = new AllowableValue("usm-json-file", "Json File Path", "The path of the JSON file containing the USM users");
+    public static final AllowableValue USM_JSON = new AllowableValue("usm-json", "Json File", "The JSON file containing the USM users");
```

Comment:
As we agreed, I'd use JSON File Content displayName for this value. (probably I'd rename the variable name too and I'd add _PATH to the variable above for better clarity.